frontend/test/metabase/scenarios/smoketest/admin.cy.spec.js

```diff
@@ -60,7 +60,7 @@ describe("metabase-smoketest > admin", () => {
       const dbFilename = "frontend/test/__runner__/empty.db";
       const dbPath = Cypress.config("fileServerFolder") + "/" + dbFilename;
       cy.findByLabelText("Connection String").type(`file:${dbPath}`);
-      cy.findByText("Next").click();
+      cy.findByText("Connect database").click();
```

Comment:
Modify this to reflect the UI change.
![image](https://user-images.githubusercontent.com/1937582/169349022-c43f9fed-3f73-4f84-b93a-b673f92a36f7.png)